a_cov_shift: false
adaptive_covariance: false
cartesian_models:
- dynamic_ST
- kinematic_ST
control_dim: 2
control_sample_std:
- 0.35
- 0.35
exp_name: mppi_tracking
friction: 1.04
init_vel: 2.0
is_sim: true
map_dir: /home/ubuntu/ese6150_ws/src/Learning-Guided-Control-MPPI/waypoints/
map_ind: 0
n_iterations: 1
n_samples: 256
n_steps: 9
norm_params:
- - 4.0
  - -2.0
- - 6.0
  - -3.0
random_seed: 527787
ref_vel: 2.0
render: 1
s_frame_max: 62.12059091197948
sim_time_step: 0.1
state_predictor: dynamic_ST
wpt_delim: ;
wpt_path: levine/levine.csv
wpt_rowskip: 3
wpt_thind: 3
wpt_vind: 5
wpt_xind: 1
wpt_yind: 2
use_mppi: True
num_scans: 1080


max_obs: 150
left: 0.5
right: 0.5
invalid_pos:  -1000.0
safety_margin: 0.15